staging/cert-manager-setup/templates/certificates.yaml

```diff
@@ -39,14 +39,16 @@ spec:
     - {{ . | quote -}}
   {{- end }}
 {{- end }}
-  keyAlgorithm: {{ .keyAlgorithm | default "rsa" }}
-  keyEncoding: {{ .keyEncoding | default "pkcs1" }}
-  keySize: {{ .keySize | default "2048" }}
+  privateKey:
+    algorithm: {{ .keyAlgorithm | default "RSA" }}
```

Comment:
out of api docs
>  allowed values are either “rsa” or “ecdsa”

are the docs wrong ? if so 🤦

Reply:
They are. Validation fails unless they are uppercase. It would have been nice if the enums contained both cases. I can convert them to uppercase to avoid validation issues with custom values.

Reply:
Again, a PR upstream would be awesome.